app/src/main/java/com/github/sdp/ratemyepfl/model/items/Course.kt

```diff
@@ -38,4 +40,5 @@ data class Course(
         private const val TAG = "Course"
```

Comment:
Is it useful? (same question for other reviewable items classes)

Reply:
No I don't think so, but I didn't code this class, so I don't want to delete it.

Reply:
Sure, maybe @AlexandreMessmer can look at it then :)